readme.md

```diff
@@ -60,9 +60,11 @@ Some good apps written with Electron.
 ## Tools
 
 - [electron-packager](https://github.com/maxogden/electron-packager) - Package and distribute your Electron app for OS X, Linux and Windows (.app, .exe, etc).
+- [electron-builder](https://github.com/loopline-systems/electron-builder) - Build installers for Windows and OS X from Windows, Linux, or OS X.
 - [electron-prebuilt](https://github.com/mafintosh/electron-prebuilt) - Install prebuilt Electron binaries for command-line use using npm.
 - [electron-rebuild](https://github.com/paulcbetts/electron-rebuild) - Rebuild native io.js modules against the currently installed Electron version.
 - [electron-gh-release](https://github.com/jenslind/electron-gh-releases) - Electron auto-update by releasing on GitHub.
+- [electron-updater](https://github.com/evolvelabs/electron-updater) - A cross platform auto-updater, leveraging NPM to deploy updates.
```

Comment:
I kind of wanted to stress the "cross platform"-ness of it since some of the other solutions I've seen (squirrel based) required separate solutions for different platforms. I lowercased npm though.